mergin/client_pull.py

```diff
@@ -203,7 +213,7 @@ def download_project_is_running(job):
             traceback_lines = traceback.format_exception(type(exc), exc, exc.__traceback__)
             job.mp.log.error("Error while downloading project: " + "".join(traceback_lines))
             job.mp.log.info("--- download aborted")
-            job.failure_log_file = _cleanup_failed_download(job.directory, job.mp)
+            job.failure_log_file = _cleanup_failed_download(job.directory.name, job.mp)
```

Comment:
looks like we don't need to pass the temp directory name to _cleanup_failed_download(), given that it will be auto-deleted when the temporary directory object gets out of scope?

also _cleanup_failed_download() probably should not call shutil.rmtree() if the deletion happens automatically